sources/model/entity/PendingUser.ts

```diff
@@ -0,0 +1,68 @@
+import {Column, Entity, Index, PrimaryGeneratedColumn} from "typeorm";
+import { Encrypt } from "../../utils/encrypt";
+import { ApiException } from "../../utils/apiException";
+
+@Entity("PendingUser")
+export class PendingUser {
+
+    @PrimaryGeneratedColumn()
+    id: number;
+
+    @Column({length: 31})
+    @Index({unique: true})
+    pseudo: string;
+
+    @Column()
+    @Index({unique: true})
+    email: string;
+
+    @Column({name: "ciphered_password"})
+    cipheredPassword: string;
+
+    @Column()
+    @Index({unique: true})
+    validationUuid: string;
+
+    @Column({name: "created_at", type: "timestamp with time zone", default: () => "CURRENT_TIMESTAMP"})
+    createdAt: Date;
+
+    @Column({name: "updated_at", type: "timestamp with time zone", default: () => "CURRENT_TIMESTAMP"})
+    updatedAt: Date;
+
+
+    public constructor(pseudo: string, email: string, ciphered_password: string, validation_uuid: string) {
+        this.pseudo = pseudo;
+        this.email = email;
+        this.cipheredPassword = ciphered_password
+        this.validationUuid = validation_uuid;
+
+    }
+
+    // Validators
+    private static pseudo_regex = new RegExp(/^[a-zA-Z0-9_-]{2,31}$/);
+    private static email_regex = new RegExp(/\w+(?:\.\w+)*@\w+(?:\.\w+)+/);
```

Comment:
This creates a duplicate with the one in `User`. A `utils/validators/EmailValidator.ts` would be nice for those